src/Sarif.Viewer.VisualStudio/SarifViewerPackage.cs

```diff
@@ -98,7 +100,7 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
         {
             await base.InitializeAsync(cancellationToken, progress);
```

Comment:
should we call `.ConfigureAwait`?

Reply:
Fixed.

---
In reply to: [538267062](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538267062) [](ancestors = 538267062)